### PR TITLE
e2e test: error message on missing kapp e2e ns suggests how to fix

### PR DESCRIPTION
**Pull request by joe-kimmel-vmw**

### CODE REVIEW

File: test/e2e/env.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func (e Env) Validate(t *testing.T) {
 	errStrs := []string{}
 
 	if len(e.Namespace) == 0 {
-		errStrs = append(errStrs, "Expected Namespace to be non-empty")
+		errStrs = append(errStrs, "Expected Namespace to be non-empty (Hint: set ENV var KAPP_E2E_NAMESPACE")
```
Comment (100mik) [inline test/e2e/env.go]: Should we specify that the value should be set to `kapp-test`?
This value is hard-coded in a bunch of test cases if not most of them.

**Review by 100mik — CHANGES_REQUESTED** [review]

File: test/e2e/env.go (hunk 1 above)
Comment (joe-kimmel-vmw) [reply test/e2e/env.go]: sgtm;   I added an `=kapp-test`

File: test/e2e/env.go [hunk 2]
```diff
@@ -34,7 +34,7 @@ func (e Env) Validate(t *testing.T) {
 	errStrs := []string{}
 
 	if len(e.Namespace) == 0 {
-		errStrs = append(errStrs, "Expected Namespace to be non-empty")
+		errStrs = append(errStrs, "Expected Namespace to be non-empty (Hint: set ENV var KAPP_E2E_NAMESPACE=kapp-test")
```
Comment (100mik) [inline test/e2e/env.go]: Super nitty 
```suggestion
		errStrs = append(errStrs, "Expected Namespace to be non-empty (Hint: set ENV var KAPP_E2E_NAMESPACE=kapp-test)")
```

Comment (100mik) [reply test/e2e/env.go]: (Missing closing bracket for hint)

**Review by 100mik — APPROVED** [review]
Thaks for this! LGTM 🚀

**Review by sethiyash — APPROVED** [review]

**Pull request merged**